components/arm/arm.go

```diff
@@ -373,20 +381,9 @@ func Plan(ctx context.Context, logger golog.Logger, a Arm, dst spatialmath.Pose)
 	return motionplan.PlanFrameMotion(ctx, logger, dst, armFrame, armFrame.InputFromProtobuf(jp), defaultArmPlannerOptions, nil)
 }
 
-// GoToWaypoints will visit in turn each of the joint position waypoints generated by a motion planner.
-func GoToWaypoints(ctx context.Context, a Arm, waypoints [][]referenceframe.Input) error {
-	for _, waypoint := range waypoints {
-		err := ctx.Err() // make sure we haven't been cancelled
-		if err != nil {
-			return err
-		}
-
-		err = a.GoToInputs(ctx, waypoint)
-		if err != nil {
-			return err
-		}
-	}
-	return nil
+// AllWaypoints will visit in turn each of the joint position waypoints generated by a motion planner.
+func AllWaypoints(ctx context.Context, a Arm, waypoints [][]referenceframe.Input) error {
```

Comment:
I don't think there is a point to having this function anymore if it just calls a publicly accessible function